packages/ui/react-ui-theme/src/config/colors.ts

```diff
@@ -225,6 +225,10 @@ export const semanticColors = {
     light: physicalColors.neutral['700'],
     dark: physicalColors.neutral['300'],
   },
+  error: {
```

Comment:
@thure does this already exist somewhere?

Reply:
There is a fragment `messageValence` that could be refactored to use semantic tokens, however I have a better system ready to go with `@ch-ui/tokens` which would make both of these approaches obsolete. Basically this adds tech debt but not much.